docs/sources/enterprise/query-caching.md

```diff
@@ -21,20 +21,27 @@ The following cache backends are available: in-memory, Redis, and Memcached.
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
-## Enable query caching
+## Enable and configure query caching
```

Comment:
Somewhere we need to call out the required permissions to perform these tasks. For example, you need to be an Organization admin with permissions to edit the data source. Can Team Admins edit these options as well? What about Editors with editors_can_admin enabled?

Reply:
Looks like this still needs to be added, @mjseaman

Reply:
Resolved in latest commit.